operator-linebreak - config update

Nostrus opened the pull request.

Nostrus committed:
Added new rule

camilleriluke reviewed (changes requested):
The rule is already present, could you please update the already existing one? 

Also please have a look at https://github.com/Casumo/eslint-config-casumo#contributing and follow the contribution guide

Nostrus commented:
Will do so, thanks for pointing out the rules, folowing them now. 
However, `npm test` fails for me at 
`Failed at the eslint-config-casumo@1.1.0 find-new-eslint-rules:off script 'eslint-find-rules -u ./configurations/off.js'.`

`Tell the author that this fails on your system:
 eslint-find-rules -u ./configurations/off.js`

Similar for
`Tell the author that this fails on your system: npm-run-all --parallel find-new-eslint-rules:*`

Nostrus committed:
feat: Updated operator-linebreak rule

Added rule to force usage of operators at the end of lines in case of multi-line operations

camilleriluke commented on rules/eslint/stylistic-issues/on.js:
```diff
@@ -64,7 +64,7 @@ module.exports = {
         'one-var': [2, { var: 'always', let: 'never', const: 'never' }],
         'operator-assignment': 0,
-        'operator-linebreak': 0,
+        'operator-linebreak': ['error', 'after'],
```
Ideal this should confirm to the other rules and use integers to denote the error type [like so](http://eslint.org/docs/user-guide/configuring#configuring-rules): `'operator-linebreak': [2, 'after'],`

Nostrus replied:
Makes sense, pushed fix.

camilleriluke commented:
> However, npm test fails for me at
> Failed at the eslint-config-casumo@1.1.0 find-new-eslint-rules:off script 'eslint-find-rules -u ./configurations/off.js'.

Strange. I do not get this. Could you please tell me your node, npm and/or yarn versions please?

Nostrus committed:
refactor: Use integer for rule instead of string

Nostrus commented:
node v7.2.1
npm 4.0.5

camilleriluke commented:
Also can you please link the full error log.

Nostrus commented:
Sure, attached.
[errors.txt](https://github.com/Casumo/eslint-config-casumo/files/763545/errors.txt)

camilleriluke commented:
One last question: The lines before the error are similar to the following?

```
unused rules

no-multi-assign                    prefer-promise-reject-errors       template-tag-spacing
```

Nostrus commented:
Yes, i have these:
```
unused rules

no-multi-assign                   prefer-promise-reject-errors
```

camilleriluke commented:
Ok so those are legitimate errors.

Meaning that our presets are missing configs for those rules.

There are open issues for the new rules (#7, #8, and #9) so we can get them up sorted out.

Meanwhile I think this PR is good to merge.
